contrib/cray/bin/run_libfabric_pipeline

```diff
@@ -228,6 +237,22 @@ $BATS_INSTALL_PATH/bats $@ -t contrib/cray/bats/smoketests.bats | tee smoketests
 section_end 'smoke tests'
 fi
 
+if $IMBTEST ; then
+section_start 'imb tests'
+## Start IMB Tests
+$BATS_INSTALL_PATH/bats $@ -t contrib/cray/bats/imb.bats | tee imb.tap
+## End IMB Tests
+section_end 'imb tests'
+fi
+
+if $OMBTEST ; then
+section_start 'omb tests'
+## Start OMB Tests
+$BATS_INSTALL_PATH/bats $@ -t contrib/cray/bats/omb.bats | tee omb.tap
+## End OMB Tests
+section_end 'omb tests'
```

Comment:
The MPI section below should probably be removed since it is being expanded into imb and omb here.

Reply:
There are a couple of issues here.  First, MPI should not be run because mpi.bats is missing.  Second, there is a white space error in line 314 which causes SF to terminate and exit the test, which in turn causes MPI not to run (or fail in this case).
The white space problem will be fixed and the MPI section will be removed.

Reply:
Sounds good.